ts_src/psetv2/creator.ts

```diff
@@ -54,18 +52,21 @@ export class CreatorInput {
 export class CreatorOutput {
   asset: string;
   amount: number;
-  address?: string;
+  script?: Buffer;
+  blindingPublicKey?: Buffer;
```

Comment:
what if we keep also the address? The address is super handy when one wants to add  a "std" output, while script and blinding key become useful for adding dummy or burn outputs. We just need to make sure that address and script+blindKey are "mutually exclusive"

Reply:
Which means making script beocme `scriptOrAddress?: Buffer | string` and then check the number of parameters and type to determine what is being passed?

Reply:
I don't really like, complicates a bit the code.

Currently, users of Psbt are used to passing the script always, so not aware of any code breaking. Since we expose the toOutputScript and fromConfidential utils, it's fine to lower our complexity a bit?

Reply:
I'm trying to see this from the user POV.

Being able to specify the address of an output instead of script + blindKey is something I wanted even when using psetv0. But for that module we inherited the design from bitcoinjs and didn't change the existing methods beside adding liquid stuff.

But I agree with you that it's fine to lower our complexity a bit, so why not supporting both cases?

Reply:
Added in https://github.com/vulpemventures/liquidjs-lib/pull/59/commits/af0a0a9eb72ffd8682e81f39359df4ef02c4a6f4